Frontend/ui-customer/src/Screens/OnBoardingFlow/EnterMobileNumberScreen/Controller.purs

```diff
@@ -136,12 +138,12 @@ eval (MobileNumberEditTextAction (MobileNumberEditorController.TextChanged id va
     _ <- if length value ==  mobileNumberMaxLength state.data.countryObj.countryShortCode then do
             pure $ hideKeyboardOnNavigation true
             else pure unit
-    let isValidMobileNumber = mobileNumberValidator state.data.countryObj.countryCode state.data.countryObj.countryShortCode value 
-    let newState = state { props = state.props { isValidMobileNumber = isValidMobileNumber
-                                        , btnActiveMobileNumber = if isValidMobileNumber then true else false
+    let validatorResp = mobileNumberValidator state.data.countryObj.countryCode state.data.countryObj.countryShortCode value 
+    let newState = state { props = state.props { isValidMobileNumber = validatorResp == MVR.ValidPrefix || validatorResp == MVR.Valid
```

Comment:
create functions to get `isValidMobileNumber` and `btnActiveMobileNumber` as we are doing the same in two places.